lib/pf/util.pm

```diff
@@ -1095,6 +1097,44 @@ sub strip_username {
     return $username;
 }
 
+sub send_email {
+    my ($smtp_server, $from, $to, $subject, $template, %info) = @_;
+    my $logger = get_logger;
+    my %options;
+    $options{INCLUDE_PATH} = "$conf_dir/templates/";
+    $options{ENCODING} = "utf8";
+    
+    my $import_succesfull = try { require MIME::Lite::TT; };
```

Comment:
I know you just copied and pasted this code.
But MIME::Lite::TT should be included as a dependency.